week-5-practise/js/main.js

```diff
@@ -0,0 +1,66 @@
+//// PART-1 ////
+
+var placeToJumbotron = document.querySelector(".jumbotron");
+var placeToDonateABike = document.querySelector(".btn-primary"+".btn-lrg");
+var placeToVolunteer = document.querySelector(".btn-secondary");
+
+var blueButton = document.querySelector("#blueBtn");
+blueButton.addEventListener("click", blueButtonFunction);
+function blueButtonFunction() {
+    placeToJumbotron.style.backgroundColor = "#588fbd";
+    placeToDonateABike.style.backgroundColor = "#ffa500";
+    placeToVolunteer.style.backgroundColor = "black";
+    placeToVolunteer.style.color = "white";   
+}
+
+var orangeButton = document.querySelector("#orangeBtn");
+orangeButton.addEventListener("click", orangeButtonFunction);
+function orangeButtonFunction() {
+    placeToJumbotron.style.backgroundColor = "#f0ad4e";
+    placeToDonateABike.style.backgroundColor = "#5751fd";
+    placeToVolunteer.style.backgroundColor = "#31b0d5";
+    placeToVolunteer.style.color = "white";
+}
+
+var greenButton = document.querySelector("#greenBtn");
+greenButton.addEventListener("click", greenButtonFunction);
+function greenButtonFunction() {
+    placeToJumbotron.style.backgroundColor = "#87ca8a";
+    placeToDonateABike.style.backgroundColor = "black";
+    placeToVolunteer.style.backgroundColor = "#8c9c08";
+}
+
+////////// PART-2 /////
+
+var formTag = document.querySelector("form");
+formTag.lastElementChild.addEventListener("click", submitButtonFunction); 
+event.preventDefault();
+function submitButtonFunction() {
+    var emailInput = document.querySelector("#exampleInputEmail1");
+    var nameInput = document.querySelector("#example-text-input");
+    var describeTextInput = document.querySelector("#exampleTextarea");
+    if ((emailInput.value.length === 0) || (!emailInput.value.includes("@"))) {
+        emailInput.style.backgroundColor = "red";
+    } else{
+        emailInput.style.backgroundColor = "white";
+    }
+    if (nameInput.value.length === 0) {
+        nameInput.style.backgroundColor = "red";
+    } else{
+        nameInput.style.backgroundColor = "white";
+    }
+    if (describeTextInput.value.length === 0) {
+        describeTextInput.style.backgroundColor = "red";
+    } else{
+        describeTextInput.style.backgroundColor = "white";
+    }
+    if ((emailInput.value.length !== 0) && (emailInput.value.includes("@")) && (nameInput.value.length !== 0) && (describeTextInput.value.length !== 0)){
```

Comment:
This works perfectly fine. As a best practice, when you have more than one negative condition in an if statement, you would work with a negative group of conditions instead. This has the exact same outcome as your line of code but should be easier to read/comprehend:
```
iff (!(emailInput.value.length === 0 || !emailInput.value.includes("@") || nameInput.value.length === 0 || describeTextInput.value.length === 0)){
```
If this doesn't make a lot of sense don't worry and just ignore this comment.